worker-languagedetection-container/pom.xml

```diff
@@ -161,6 +165,30 @@
                     <!--<debugForkedProcess>true</debugForkedProcess>-->
                 </configuration>
             </plugin>
+            
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.4.2</version>
+                <executions>
+                    <execution>
+                        <id>download-target-file</id>
```

Comment:
Removing this and including the model in the base image will reduce build time of this image.

Reply:
Happy we do this in a future ticket.

Reply:
It was already working by the time of the comment so leaving this as it is.